apps/ui-mobile/src/app/screens/service-mode-screen.tsx

```diff
@@ -31,6 +63,14 @@ export const ServiceModeScreen = () => {
   );
 };
 
+const startModules = () => {
+  NativeModules.MetaWearModule.startMetaWearModules()
+}
+
+const stopModules = () => {
+  NativeModules.MetaWearModule.stopMetaWearModules()
+}
```

Comment:
mała drobnostka, na poczatku pliku mozna destrukturyzować `NativeModules` w ten sposób `const { MetaWearModule } =  NativeModules as  NativeModulesInterface` i wtedy sie pozbedziemy prefixów w całym pliku

Reply:
to to tez do poprawy

Reply:
Done